test/Microsoft.AspNet.Razor.Runtime.Test/TagHelpers/DefaultTagHelperContentTest.cs

```diff
@@ -301,7 +301,6 @@ public void Append_WithTagHelperContent_MultipleAppends()
 
             // Assert
             Assert.Equal(expected, copiedTagHelperContent.GetContent());
-            Assert.Equal(new[] { text1, text2 }, copiedTagHelperContent.ToArray());
```

Comment:
Because it is an implementation detail of `BufferedHtmlContent` which has been verified in its own tests. Since `TagHelperContent` is no longer an `IEnumerable` I have removed it.